.eslintrc

```diff
@@ -1,31 +1,71 @@
 {
   "root": true,
-  "parser": "babel-eslint",
-  "extends": ["airbnb-base", "prettier", "plugin:import/errors", "plugin:import/warnings"],
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "prettier"
+  ],
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true
+    },
+    "ecmaVersion": 2020,
+    "sourceType": "module"
+  },
+  "plugins": ["@typescript-eslint", "prettier"],
+  "env": {
+    "browser": true,
+    "node": true,
+    "es6": true
+  },
   "rules": {
     "prettier/prettier": "error",
-    "import/prefer-default-export": "off",
-    "import/no-extraneous-dependencies": [
+    "@typescript-eslint/naming-convention": [
       "error",
       {
-        "devDependencies": true,
-        "optionalDependencies": true
+        "format": ["PascalCase"],
+        "selector": "typeLike",
+        "custom": {
+					"regex": "^I[A-Z]",
```

Comment:
@ambrauer I used this rule, because it was added in tslint rules:
https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/naming-convention.md#enforce-that-interface-names-do-not-begin-with-an-i